test/integration-new/cni-upgrade-downgrade/host_networking_test.go

```diff
@@ -0,0 +1,99 @@
+package cni_upgrade_downgrade
```

Comment:
Can we include custom networking and PD mode test with upgrade/downgrade?

Reply:
For Custom networking we create a new node group but for PD mode we should be able to run against existing cluster. Could you elaborate on the steps for how exactly we want the run the PD tests with upgrade/downgrade scenario.

Reply:
Have added PD traffic test, this is the flow
apply initial manifest -> Set PD -> Run traffic test -> apply target manifest with PD enabled -> Run traffic test -> roll back to initial manifest -> reset PD.

Reply:
Thank you!